WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -273,11 +282,15 @@ protected void onPause() {
 
     @Override
     protected void onSaveInstanceState(Bundle outState) {
-        outState.putInt(SAVED_NAV_POSITION, mNavPosition);
         outState.putInt(SAVED_WP_LOGIN_STATE, mResultCode);
         outState.putInt(ARG_LOCAL_TABLE_BLOG_ID, mLocalBlogID);
         outState.putSerializable(SAVED_STATS_TIMEFRAME, mCurrentTimeframe);
         outState.putString(SAVED_STATS_REQUESTED_DATE, mRequestedDate);
+        outState.putIntArray(SAVED_STATS_SCROLL_POSITION,
+                new int[]{
+                        mOuterScrollView.getScrollX(),
```

Comment:
This isn't going to be 100% accurate when restored after device rotation, unless every view is the same height regardless of orientation. It shouldn't be wildly off so this may not be worth the effort, but it might be more accurate to store which child view (stats section) is closest to the top along with how far it's scrolled from the top.